src/app/app.component.ts

```diff
@@ -84,6 +85,7 @@ export class AppComponent {
       if (event instanceof NavigationStart) {
         reportNavigationStart(event.url)
         this.currentRoute = event.url
+        console.log('currentRoute', this.currentRoute)
```

Comment:
@DanielPalafox could you please remove this console log?

Reply:
Done!